site/pages/api/publish-edit.api.ts

```diff
@@ -84,6 +90,13 @@ const publishEdit = async (req: NextApiRequest, res: NextApiResponse) => {
                   session.name,
               );
 
+        if (
+            validatedDisruptionBody.data.socialMediaPosts &&
+            validatedDisruptionBody.data.socialMediaPosts.length > 0 &&
+            canPublish(session)
+        ) {
+            await publishToHootsuite(validatedDisruptionBody.data.socialMediaPosts, session.orgId);
```

Comment:
As discussed - create a media post as a staff user and sent the disruption for review. Logged in as org admin user and created another media post. Then edited the first media post added by staff user. At this stage there were two media posts with pending status.

On clicking publish disruption only the second media post was set to successful while the other media post was set to rejected. Expected result would be that both are set to Successful

Reply:
done